llvm/lib/CodeGen/GlobalISel/Utils.cpp

```diff
@@ -301,7 +302,8 @@ std::optional<APInt> llvm::getIConstantVRegVal(Register VReg,
 }
 
 std::optional<int64_t>
-llvm::getIConstantVRegSExtVal(Register VReg, const MachineRegisterInfo &MRI) {
+llvm::getIConstantVRegSExtVal(Register VReg, const MachineRegisterInfo &MRI,
+                              bool LookThroughInstrs = false) {
```

Comment:
You should pass `LookThroughInstrs`to `getIConstantVRegVal`.

Reply:
Yes, thats important!